internal/api/database/model/dao.go

```diff
@@ -21,6 +21,7 @@ import (
 )
 
 type Query interface {
+	GetMetadataOnlyQueryParam() bool
```

Comment:
Let's open the debate that we all expect: should it be ``metadata_only`` or something else? 

Food for think: 
What if the default was to actually give only the  metadata to prevent from dumb requests. You could have full=true for example in all the queries that actually want details

Reply:
partial, full, skip_spec, ignore_spec, ...
Depends on the the default behavior too, "full=true" will not work with the current implem

Reply:
Yes that's what I'm saying. Changing the default behaviour to by default having full=false would in a sense give a message that if you use full=true you don't use default and don't complain if it fails or take time to answer, or is paginated.

Reply:
When you get the list of data with no query parameter, you won't expect a partial response data.
What we can do is to provide a pagination so you cannot get everything excepting if you explicitly say it.

`metadata_only` is at least explicitly saying you will only get the metadata and nothing else.

Reply:
but before including a pagination, I want to make everything to be able to get all data without crashing Perses.

Reply:
Okay, just wanted to make sure we explore all the possibilities. 

I think I'd tend to be in favor of an API that **by default** gives everything and light than something very detailed and potentially paginated in the future. But at the end as long as the light mode is available I'm good with any of the default behaviour.